recipes/face-recognition/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "face_recognition" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5e5efdd1686aa566af0d3cc1313b131e4b197657a8ffd03669e6d3fad92705ec
+
+build:
+  number: 0
```

This looks like it can be `noarch: python`?
```suggestion
  number: 0
  noarch: python
```

---

Please make this noarch